src/libraries/System.Private.CoreLib/src/System/Threading/SemaphoreSlim.cs

```diff
@@ -211,8 +211,7 @@ public void Wait(CancellationToken cancellationToken)
         /// <returns>true if the current thread successfully entered the <see cref="SemaphoreSlim"/>;
         /// otherwise, false.</returns>
         /// <exception cref="ArgumentOutOfRangeException"><paramref name="timeout"/> is a negative
-        /// number other than -1 milliseconds, which represents an infinite time-out -or- timeout is greater
```

Comment:
The doc comments are not the official sources for the docs. Could you please submit PR to https://github.com/dotnet/dotnet-api-docs/ to add a note that the ArgumentOutOfRangeException is no longer thrown for timeout greater than int.MaxValue in .NET 10?

Reply:
PR here https://github.com/dotnet/dotnet-api-docs/pull/11677